openapi/index.yaml

```diff
@@ -0,0 +1,133 @@
+swagger: "2.0"
+info:
+  version: 0.1.0
+  title: PagoPA RT Manager
+  contact:
+    name: PagoPA spa
+    url: https://forum.italia.it/c/progetto-io
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
+    Documentation of the IO Function API here.
+host: api.cd.italia.it
```

Comment:
remmber to change this hostname

Reply:
Fixed in 05b02f7. But the production value must be used when available.